app/models/work.rb

```diff
@@ -0,0 +1,26 @@
+class Work < ApplicationRecord
+  # relationships
+  has_many :votes, dependent: :destroy
+  has_many :users, through: :votes
+
+  # validation
+  CATS = %w(album movie book)
+  validates :creator, :publication_year, :title, :category, presence: true
+  validates :title, length: { in: 1..200 }
+  validates :title, uniqueness: true
+  validates :category, inclusion: { in: CATS }
+  # pub year must be number in year range 0 - 2018
+  validates :publication_year, numericality: true, inclusion: 0..2018
+
+  def self.top?(category)
```

Comment:
the question mark here has a semantic meaning that you aren't using correctly. A question mark at the end of a method signature implies that it returns a boolean.